app/Http/Controllers/ContactUsController.php

```diff
@@ -2,12 +2,41 @@
 
 namespace App\Http\Controllers;
 
-use Illuminate\Http\Request;
+use App\Http\Requests\ContactUsRequest;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Mail\Message;
 
 class ContactUsController extends Controller
 {
     public function contact()
     {
-        return view('home.contact');
+        return view('contacts.contactUs');
     }
-}
\ No newline at end of file
+
+    public function send(ContactUsRequest $request): RedirectResponse
+    {
+
+        // dd($request->validated());
+        $data = $request->validated();
+        \Log::debug('test', $data);
+
+        \Mail::send(
+            'emails.contactUs',
+            [
+                'firstname' => $data['firstname'],
+                'lastname' => $data['lastname'],
+                'email' => $data['email'],
+
+                'messageText' => $data['message'],
+
+            ],
+            function (Message $message) use ($data) {
+                $message->subject('message from ' . $data['email']);
+                $message->to('tech@baloon.app');
```

Comment:
please adjust to and from for your project values.

Reply:
changed